docs/tutorial/Build-the-bundle.md

```diff
@@ -1,24 +1,11 @@
-1.  <span>[Developer](index)</span>
-2.  <span>[Creating a Tweet Field Type](Creating-a-Tweet-Field-Type)</span>
+# Build the bundle
 
-<span id="title-text"> Developer : Build the bundle </span>
-===========================================================
-
-
-
-<span>FieldTypes, like any other eZ Platform extensions, must be provided as Symfony 2 bundles. This chapter covers the creation and organization of this bundle.</span>
+FieldTypes, like any other eZ Platform extensions, must be provided as Symfony 2 bundles. This chapter covers the creation and organization of this bundle.
 
 First, you will see how to generate the skeleton for a standard Symfony 2 bundle using the console: [Create the bundle](Create-the-bundle). Then, you can learn about the suggested structure for storing a Field Type inside a bundle: [Structure the bundle](Structure-the-bundle).
 
- 
-
 ------------------------------------------------------------------------
 
- 
-
- <span class="char" title="Leftwards Black Arrow">⬅</span> Previous: [Introduction to "Creating a Tweet Field Type" tutorial](Creating-a-Tweet-Field-Type)
-
-Next: <span class="confluence-link" style="text-align: right;" title="Black Rightwards Arrow">[Create the bundle](Create-the-bundle) ➡</span>
-
-
+⬅ Previous: [Introduction to "Creating a Tweet Field Type" tutorial](Creating-a-Tweet-Field-Type)
```

Comment:
I think for the time being yes, but we'll have to remove it once we prepare the files for generation with RTD